src/common/http/http.cc

```diff
@@ -621,6 +623,71 @@ response_t HttpImpl::Get(
   }
 }
 
+response_t HttpImpl::SimpleGet(
```

Comment:
We don't have transport socket abstraction (raw or tls). This is why we must add another redundant simple HTTP request function. We should introduce transport socket abstraction to clean up these codes.